iam/api-client/src/main/java/com/google/iam/snippets/ServiceAccountKeys.java

```diff
@@ -15,47 +15,54 @@
 
 package com.google.iam.snippets;
 
+// [START iam_create_key]
+// [START iam_list_keys]
+// [START iam_delete_key]
 import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
 import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
 import com.google.api.client.json.jackson2.JacksonFactory;
 import com.google.api.services.iam.v1.Iam;
 import com.google.api.services.iam.v1.IamScopes;
-import com.google.api.services.iam.v1.model.CreateServiceAccountKeyRequest;
 import com.google.api.services.iam.v1.model.ServiceAccountKey;
 import java.io.IOException;
 import java.util.Collections;
+// [END iam_create_key]
+// [END iam_list_keys]
+// [END iam_delete_key]
+
+// [START iam_create_key]
+import com.google.api.services.iam.v1.model.CreateServiceAccountKeyRequest;
+// [END iam_create_key]
+
+// [START iam_list_keys]
 import java.util.List;
+// [END iam_list_keys]
+
+// [START iam_create_key]
+// [START iam_list_keys]
+// [START iam_delete_key]
 
 public class ServiceAccountKeys {
 
   private final Iam service;
 
   public ServiceAccountKeys() throws Exception {
-    GoogleCredential credential =
-        GoogleCredential.getApplicationDefault()
-            .createScoped(Collections.singleton(IamScopes.CLOUD_PLATFORM));
-
-    service =
-        new Iam.Builder(
-                GoogleNetHttpTransport.newTrustedTransport(),
-                JacksonFactory.getDefaultInstance(),
-                credential)
-            .setApplicationName("service-account-keys")
-            .build();
+    GoogleCredential credential = GoogleCredential.getApplicationDefault()
+        .createScoped(Collections.singleton(IamScopes.CLOUD_PLATFORM));
+
+    service = new Iam.Builder(GoogleNetHttpTransport.newTrustedTransport(), JacksonFactory.getDefaultInstance(),
```

Comment:
I don't think that matches our [line breaking rules](https://google.github.io/styleguide/javaguide.html#s4.5.1-line-wrapping-where-to-break)